11-Sophia-Pache/wk02/5-fri/atm/js/atm.js

```diff
@@ -0,0 +1,94 @@
+        let balance = $('#checking-balance').text();
+        balance = (balance).slice(1);
+        const newBalance = Number(balance) - Number(withdrawVal);
+        let savingsBalance = $('#savings-balance').text();
+        savingsBalance = (savingsBalance).slice(1);
+        if (newBalance >= 0) { // this conditonal makes sure you don't withdraw over your balance value
+            $('#checking-balance').html(`$ ${newBalance}`);
+            if (newBalance === 0) {
+                $('#checking .balance').addClass('zero'); // this changes the background to red if your balance is equal to 0
+            }
+        } else if (savingsBalance >  0) {
+            $('#savings-balance').html(`$ ${newBalance}`);
+            overdraftSavings(newBalance);
+            // the new savings balance that's negative
+            // add the negative value back to other account
+            // set account to zero
+        }
+    })
+})
+
+
+$(document).ready(function () { // this function will allow the user to DEPOSIT money in savings based on their input, adding the original value and the input value 
+    $('#savings-deposit').on('click', function () {
+        const depositVal = $('#savings-amount').val();
+        const balance = $('#savings-balance').text();
+        const number = (balance).slice(1);
+        const depositTotal = Number(number) + Number(depositVal);
+        $('#savings-balance').html(`$ ${depositTotal}`);
+    })
+})
+
+$(document).ready(function () { // this function will allow the user to WITHDRAW money from savings based on their input, subtracting the input value from the original value 
+    $('#savings-withdraw').on('click', function () {
+        const withdrawVal = $('#savings-amount').val();
+        let balance = $('#savings-balance').text();
+        balance = (balance).slice(1);
+        const newBalance = Number(balance) - Number(withdrawVal);
+        let checkingBalance = $('#checking-balance').text();
+        checkingBalance = (checkingBalance).slice(1);
+        if (newBalance >= 0) { // this conditonal makes sure you don't withdraw over your balance value
+            $('#savings-balance').html(`$ ${newBalance}`);
+            if (newBalance === 0) {
+                $('#savings .balance').addClass('zero'); // this changes the background to red if your balance is equal to 0
+            }
+        } else if (checkingBalance >  0) {
+            $('#savings-balance').html(`$ ${newBalance}`);
+            overdraftChecking(newBalance);
+            // the new savings balance that's negative
+            // add the negative value back to other account
+            // set account to zero
+        }
+    })
+})
+
+
+
+// write function that given overdraw amount, withdraw from checking / savings if theres enough money
+const overdraftChecking = function (newBalance) {
+    let checkingBalance = $('#checking-balance').text();
+    checkingBalance = (checkingBalance).slice(1);
```

Comment:
```suggestion
    checkingBalance = checkingBalance.slice(1);
```
Just a code slight style change here, the round brackets aren't necessary.